medcat/cat.py

```diff
@@ -510,8 +517,19 @@ def unlink_concept_name(self, cui, name, preprocessed_name=False):
         for c in cuis:
             self.cdb.remove_names(cui=c, names=names)
 
-    def add_and_train_concept(self, cui, name, spacy_doc=None, spacy_entity=None, ontologies=set(), name_status='A', type_ids=set(),
-                              description='', full_build=True, negative=False, devalue_others=False, do_add_concept=True):
+    def add_and_train_concept(self,
+                              cui: str,
+                              name: str,
+                              spacy_doc: Optional[Doc] = None,
+                              spacy_entity: Optional[List[Token]] = None,
```

Comment:
Possible that this can also be a span: https://spacy.io/api/span

Reply:
Done.